test/tools/WebListener/WebListener.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.0-rc1-30676" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.0-rc1-final" />
```

Comment:
Is `test/tools/WebListener/NuGet.config` still necessary? Per our discussion at https://github.com/PowerShell/PowerShell/pull/6718/files#r185298874, it would not be needed when the packages are available on nuget.org.

Reply:
You're right, no longer needed.  Will remove.